pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/nsx_nsx-v-eos/guide.en-gb.md

```diff
@@ -26,8 +26,7 @@ To identify your usage of the NSX-V feature in your Hosted Private Cloud environ
 
 #### I do not use any of the NSX-V features
 
-If you do not use any of the above features, we will deactivate NSX-V starting 1st December 2023. 
-You will be notified when the deactivation process is available.
+We will disable NSX-V from 01 December 2023 if you are not using any NSX-V features.
```

Comment:
What about line 75?

Reply:
Good call, i didn't notice that. I will delete it too